frontend/src/components/Button/__snapshots__/SimpleButton.test.tsx.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly for primary base variant 1`] = `
+<button
+  className="inline-flex h-10 w-full items-center justify-center rounded-lg text-white px-4 py-2.5 text-center text-sm font-medium focus:outline-none focus:ring-4 focus:ring-blue-300 border bg-primary-500 dark:bg-primary-300 border-primary-500 hover:bg-primary-600 dark:hover:bg-primary-200 active:bg-primary-900 text-primary-50 dark:text-primary-700"
```

Comment:
Do we really want so precise checks? This looks like a maintenance effort on every cosmetic change? Or is it so easy to redo the snapshots or useful in some way? Can you please write a README how to do it?

If you learned something during testing from the code, please add code comments.

Reply:
These are automatically generated. Apparently you can easily regenerate them if a component was changed.